util/binfmt_misc/detect.go

```diff
@@ -55,6 +61,16 @@ func WarnIfUnsupported(pfs []string) {
 					printPlatfromWarning(p, err)
 				}
 			}
+			if p == "linux/ppc64le" {
+				if err := ppc64leSupported(); err != nil {
+					printPlatfromWarning(p, err)
```

Comment:
i guess this is unrelated to this PR but lovely typo in `printPlatfromWarning` :D